internal/delivery/http/common/routes.go

```diff
@@ -27,4 +27,11 @@ func (s *Server) makeRoutes() {
 	v1.HandleFunc("/me", hV1.CurrentUser).Methods(http.MethodGet)
 
 	v1.HandleFunc("/feed", hV1.Feed).Methods(http.MethodGet)
+
+	v1.HandleFunc("/users/{authorID:[0-9]+}/articles", hV1.UserArticles).Methods(http.MethodGet)
+
+	v1.HandleFunc("/users/{userID:[0-9]+}", hV1.GetUserInfo).Methods(http.MethodGet)
+	v1.HandleFunc("/users/{userID:[0-9]+}", hV1.UpdateUserInfo).Methods(http.MethodPut)
+
+	v1.HandleFunc("/users/{userID:[0-9]+}/changepassword", hV1.UpdatePassword).Methods(http.MethodPut)
```

Comment:
Давай слова отделять четко:
```suggestion
	v1.HandleFunc("/users/{userID:[0-9]+}/changePassword", hV1.UpdatePassword).Methods(http.MethodPut)
```